apps/drawing-tools.html

```diff
@@ -586,20 +591,33 @@
       return Microsoft.Maps.SpatialMath.Geometry.bounds(shapes);
     }
 
+
+
     /**
      * Callback to initialize map and event handlers.
      *
      * @return {void}
      */
     function getMap() {
-      Microsoft.Maps.loadModule(["Microsoft.Maps.DrawingTools", "Microsoft.Maps.GeoJson", "Microsoft.Maps.SpatialMath", "Microsoft.Maps.WellKnownText"], function() {
+      Microsoft.Maps.loadModule(["Microsoft.Maps.DrawingTools", "Microsoft.Maps.GeoJson", "Microsoft.Maps.SpatialMath", "Microsoft.Maps.WellKnownText", "Microsoft.Maps.AutoSuggest"], function() {
```

Comment:
Please move "Microsoft.Maps.AutoSuggest" to the start of the array (so that the elements are in alphabetical order).